connect/src/wormhole.ts

```diff
@@ -259,7 +259,7 @@ export class Wormhole {
    */
   async getDecimals(
     chain: ChainName,
-    token: TokenId | "native",
+    token: NativeAddress<any> | UniversalAddress | "native",
```

Comment:
any here because of some issue w/ using `PlatformName`?

Reply:
Just lazy really 😅 Updated it